packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendarHeader.tsx

```diff
@@ -103,7 +103,12 @@ export const DatepickerCalendarHeader: ComponentWithAs<'div', DatepickerCalendar
         ...unhandledProps,
       })}
     >
-      {createShorthand(Text, label, { defaultProps: () => getA11yProps('label', {}) })}
+      {createShorthand(Text, label, {
+        defaultProps: () =>
+          getA11yProps('label', {
+            weight: 'bold',
```

Comment:
Let's move that to our styles instead of using Text props

Reply:
Moved.